backend/deepchecks_monitoring/ee/api/v1/members.py

```diff
@@ -54,7 +54,7 @@ async def update_user_role(roles_schema: RoleUpdateSchema,
     if user.organization_id != current_user.organization_id:
         raise BadRequest("User doesn't exists in your organization.")
 
-    roles: t.List[Role] = await session.scalars(sa.select(Role).where(Role.user_id == user_id))
+    roles: t.List[Role] = (await session.execute(sa.select(Role).where(Role.user_id == user_id))).scalars().all()
```

Comment:
why? `session.scalars` does exactly the same thing, does not it?

Reply:
`session.scalars` body
![image](https://github.com/deepchecks/monitoring/assets/71635444/b2027bd8-32f3-411f-9293-2cb15e7ac6b7)

Reply:
it doesn't, it what made the bug